src/dmd/backend/os.c

```diff
@@ -80,7 +80,7 @@ void os_error(int line)
 #pragma noreturn(os_error)
 #endif
 
-#if _WIN32
+#if _WIN32 && !_WIN64
```

Comment:
Why are `_WIN32` and `_WIN64` both `true`?

Reply:
https://msdn.microsoft.com/en-us/library/b0084kay.aspx

> _WIN32 Defined as 1 when the compilation target is 32-bit ARM, 64-bit ARM, *x86*, or *x64*. Otherwise, undefined.

> _WIN64 Defined as 1 when the compilation target is 64-bit ARM or *x64*. Otherwise, undefined.